app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/worker/SelfCheckerWorker.kt

```diff
@@ -31,6 +32,10 @@ class SelfCheckerWorker(
             if (prefs.hasOutdatedKeyData()) {
                 LocalNotificationsHelper.showOutdatedDataNotification(context)
             }
+
+            if (InAppUpdateHelper.isObsolete()) {
```

Comment:
Přidal bych logiku, aby se notifikace nezobrazila při každém self-checku (několikrát za den), to by bylo otravné, prober tu četnost s Honzou a Honzou.